src/covid_model_seiir_pipeline/lib/ihme_deps.py

```diff
@@ -13,14 +13,26 @@ def _lazy_import_callable(module_path: str, object_name: str):
         return getattr(module, object_name)
     except ModuleNotFoundError:
         def f(*args, **kwargs):
-            raise ModuleNotFoundError(f"No module named '{module_path}'")
-        return f
+            raise ModuleNotFoundError(f"No module named '{module_path}'. Cannot find {object_name}.")
+    return f
 
 
-get_location_metadata = _lazy_import_callable('db_queries', 'get_location_metadata')
+try:
+    from db_queries import get_location_metadata
+except ModuleNotFoundError:
+    get_location_metadata = _lazy_import_callable('db_queries', 'get_location_metadata')
 
-Workflow = _lazy_import_callable('jobmon.client', 'Workflow')
-BashTask = _lazy_import_callable('jobmon.client', 'BashTask')
-ExecutorParameters = _lazy_import_callable('jobmon.client.swarm.executors.base', 'ExecutorParameters')
-DagExecutionStatus = _lazy_import_callable('jobmon.client.swarm.workflow.task_dag', 'DagExecutionStatus')
-WorkflowAlreadyComplete = _lazy_import_callable('jobmon.client.swarm.workflow.workflow', 'WorkflowAlreadyComplete')
+try:
```

Comment:
The other implementation was obfuscating the imports in a way that made static analysis fail.  Trading more verbosity here for the ability to introspect objects is an easy one, I think.